components/ipfs/translate_ipfs_uri.h

```diff
@@ -10,7 +10,10 @@
 
 namespace ipfs {
 
-bool TranslateIPFSURI(const GURL& url, GURL* new_url, const GURL& gateway_url);
+bool TranslateIPFSURI(const GURL& url,
+                      GURL* new_url,
+                      const GURL& gateway_url,
+                      bool use_subdomain);
```

Comment:
IMO using another function like `TranslateIPFSURIWithSubdomain` would be more readable (and there is only one spot where it is needed), but not a big deal

Reply:
Will consider on a future refactor, thanks!